rotkehlchen/api/v1/schemas.py

```diff
@@ -3531,3 +3532,55 @@ class FalsePositveSpamTokenSchema(Schema):
 
 class SpamTokenSchema(Schema):
     token = AssetField(expected_type=EvmToken, required=True)
+
+
+class CalendarCommonEntrySchema(Schema):
+    name = fields.String(required=True)
+    description = fields.String(load_default='')
+    counterparty = fields.String(load_default=None)
```

Comment:
Who/where checks given counterpart is a valid counterparty?

Reply:
For the history events we don't check it either. I'll add the same logic we use to provide them to validate it